modules/persistent-storage-csi-migration-automatic-ga.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-automatic-ga_{context}"]
+= Automatic migration of in-tree volumes to CSI
+
+{product-title} supports automatic and seamless migration for the following in-tree volume types to their Container Storage Interface (CSI) driver counterpart:
+
+* Azure Disk
+
+* OpenStack Cinder
+
+CSI migration for these volume types is considered Generally Available (GA), and requires no manual intervention.
+
+Upgrading from {product-title} versions 4.10, and earlier, to 4.11, or later, automatically migrates existing in-tree volumes so that they are managed by their equivalent CSI driver.
```

Comment:
We have missed some details about storageclasses from https://github.com/openshift/openshift-docs/pull/45281/files#diff-1116fd8ef55d77a7e4858d07631693b56bce7d56792872528a63173f7b3432b7R25